lib/sphinx/integration/server_pool.rb

```diff
@@ -56,11 +58,11 @@ def take_all
       private
 
       def choose(skip_servers)
-        if skip_servers.any?
-          servers =  @servers.select { |server| !skip_servers.include?(server) }
-        else
-          servers = @servers
-        end
+        servers = if skip_servers.empty?
+                    @servers
+                  else
+                    @servers.select { |server| !skip_servers.include?(server) }
```

Comment:
Style/InverseMethods: Use reject instead of inverting select.

Reply:
+1